nodejs/express-redis/app/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "express-redis",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "server": "node app.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@appsignal/express": "*",
+    "@appsignal/nodejs": "*",
+    "express": "4.17.1",
+    "pug": "*",
```

Comment:
Can we also lock this package version so that it doesn't break whenever a new package is pushed. These test apps are not meant to test compatibility of all packages.